### ISSUE TITLE
Package fails to activate

### ISSUE THREAD

**Issue by yitzchak** [thomasjo/atom-latex#293]
I got a failed to activate on Windows. This may be related to @jdily report in #69. After further investigation I determined that the Atom version was 1.8.0 with a Node version of 5.1.0. `OpenerRegistry` was failing because `path.format` expected a `base` argument and ignored the combination of `name` and `ext`. This issue was fixed after v5.1.0 of Node. Upgrading to Atom 1.10.0 fixed the issue.

**TL;DR** We may need to set a minimum Atom version >1.8

**Comment by thomasjo** [comment]
I don't really care about old versions of Atom. But the only acceptable alternative to simply ignoring this is changing the minimum version (as suggested).
